string trace_id
string trans_id
string che_id
string timestamp
string version